scripts/copy-files.js

```diff
@@ -1,13 +1,16 @@
 /* eslint-disable no-console */
+const path = require('path');
```

Comment:
We were using babel-node so we could write the source with es modules. I'm curious why you want to change this point.

Reply:
`babel-node`:
- imposes a runtime overhead
- makes debugging harder if sourcemaps are not setup properly

it enables in our case:
- es modules which don't provide any value in the node runtime

Reply:
My motivation was more: let's write the same code everywhere even if it's not the best tool for the job. The difference is negligeable. Simplicity first. I didn't know about the source map issue.

Reply:
Let's defer this to a different PR then. I would like to get rid of it completely. 

Since node 8 already has `async/await` it doesn't add any value at the moment.

Reply:
In some way, we might always need Babel. The JavaScript language is moving. Won't we need it too if we start using TypeScript?

Reply:
Browser and node are two different targets. We use our node scripts only internally so requiring something like node 10 or 12 is not that big of a deal. Doing so removes the babel requirement apart from the latest es features and there is no big value gained by using those features.

Typescript for our node scripts is somewhat overkill at the moment. I think you're mixing various concepts here: `babel-node` has nothing to with our components (or Ssr). It's currently only used to use es modules in our node scripts that provide no benefit in the node runtime.

Reply:
I have revisited a bit the implementation. Let me know what you think.